mvc/src/main/java/nextstep/mvc/view/JspView.java

```diff
@@ -1,30 +1,58 @@
 package nextstep.mvc.view;
 
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Map;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Map;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
 
 public class JspView implements View {
 
     private static final Logger log = LoggerFactory.getLogger(JspView.class);
 
     public static final String REDIRECT_PREFIX = "redirect:";
 
+    private final String viewName;
+
     public JspView(final String viewName) {
+        this.viewName = viewName;
     }
 
     @Override
     public void render(final Map<String, ?> model, final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        // todo
+        if (isRedirect()) {
+            response.sendRedirect(extractRedirectUrl());
+            return;
+        }
+        setModelAttribute(model, request);
+        forward(request, response);
+    }
+
+    private boolean isRedirect() {
+        return viewName.startsWith(JspView.REDIRECT_PREFIX);
```

Comment:
r: JspView에서 선언된 스태틱 상수이므로 JspView를 뗄 수 있지 않을까요?

그러면 상수를 private으로 지정해도될 것 같습니다!

```suggestion
        return viewName.startsWith(REDIRECT_PREFIX);
```